### ISSUE TITLE
Way to choose output buffer

### ISSUE THREAD

**Issue by AkselMeola** [bit101/go-ansi#3]
Hey, 

Looks cool, but I think you could add an option to direct the output to a custom writer. 
This way it would be possible to write to a connection buffer for example. 

Could do something like this:

```go

func Fprint(output io.Writer, col ansiColor, s ...any) {
	fmt.Fprintf(output, "\033[0;%dm", col.value)
	if col.bold {
		fmt.Fprintf(output, "\033[1m")
	}
	fmt.Fprint(output, s...)
	applySettings()
}

```

I plan to use it for a small telnet experiment. 

Thanks for the code :1st_place_medal:

**Comment by bit101** [comment]
Thanks. I'll take a look at doing that.

**Comment by bit101** [comment]
Sorry for the delay, `Fprint`, `Fprintln` and `Fprintf` are now implemented, along with some other goodies in `v1.5.0`

**Comment by bit101** [comment]
done
